src/integrationTest/java/uk/gov/hmcts/reform/sscs/service/IssueFurtherEvidenceServiceIt.java

```diff
@@ -449,28 +450,38 @@ public void appealWithAppellantAndMultipleOtherPartiesAndMultipleFurtherEvidence
 
         assertEquals("Test Rep", pdfDocumentRequest.getAllValues().get(0).getData().get("name"));
         assertEquals("609-97-template (original sender)", documentCaptor.getAllValues().get(0).get(0).getName());
-        assertEquals("evidence-document2", documentCaptor.getAllValues().get(0).get(1).getName());
-        assertEquals("evidence-document", documentCaptor.getAllValues().get(0).get(2).getName());
+        Assertions.assertThat(documentCaptor.getAllValues().get(0))
```

Comment:
You're still hardcoding indexes, you could move this up to find "Test Rep" in all the values and so on